go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/rosa
 
-go 1.20
+go 1.21
```

Comment:
@chenz4027 We do not yet currently support `1.21` for ROSA. Can you please downgrade to `1.20`

Reply:
@robpblake Yep there was an issue where local go impacts the `go mod tidy`, will downgrade it.